packages/ia-topnav/src/lib/query-handler.js

```diff
@@ -0,0 +1,7 @@
+/* istanbul ignore file */
+
+export default {
+  performQuery(query) {
+    window.location = `https://web.archive.org/web/*/${query}`;
```

Comment:
QUESTION: Does `query` need to be URL-encoded?

Reply:
Yeah, it should. Good catch. I've run it through `encodeURIComponent`